CMakeLists.txt

```diff
@@ -233,18 +233,14 @@ if (LLAMA_K_QUANTS)
 endif()
 
 if (LLAMA_CLBLAST)
-    find_package(CLBlast)
-    if (CLBlast_FOUND)
-        message(STATUS "CLBlast found")
+    # build CLBlast from source
+    add_subdirectory(../CLBlast ${CMAKE_CURRENT_BINARY_DIR}/clblast)
```

Comment:
This change is not OK.
You probably have to add some custom CMake option to be able to optionally select CLBlast from source

Reply:
Alright, will do